lib/AST/ASTStructuralEquivalence.cpp

```diff
@@ -1047,6 +1047,31 @@ static bool IsStructurallyEquivalent(StructuralEquivalenceContext &Context,
   return true;
 }
 
+/// Determine structural equivalence of two lambda classes.
+static bool
+IsStructurallyEquivalentLambdas(StructuralEquivalenceContext &Context,
+                                CXXRecordDecl *D1, CXXRecordDecl *D2) {
+  assert(D1->isLambda() && D2->isLambda() &&
+         "Must be called on lambda classes");
+
+  // Check the methods for consistency.
+  CXXRecordDecl::method_iterator Method2 = D2->method_begin(),
```

Comment:
Is it important to check all methods? The `getLambdaCallOperator` could be used too.

Reply:
Yes, that's a good point, I'll change that. Thanks!